zipkin-lens/src/components/DependenciesPage/DependenciesGraph.tsx

```diff
@@ -94,10 +94,6 @@ const reactSelectStyles = {
     ...base,
     cursor: 'pointer',
   }),
-  clearIndicator: (base: CSSProperties) => ({
-    ...base,
-    cursor: 'pointer',
-  }),
```

Comment:
I think this code is alive.

Without this code the cursor does not change when hovering over the clear button.

![without](https://user-images.githubusercontent.com/19551419/92927835-32e04480-f479-11ea-8979-b67d12775f24.gif)

With this code: 

![with](https://user-images.githubusercontent.com/19551419/92927901-4c818c00-f479-11ea-9f9d-3d7df6cb91f7.gif)

Reply:
intellij analysis is imperfect :) I will put a comment that this is "not quite dead"